VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -164,10 +174,14 @@ def log_with_no_try_except(self, message, level='Info'):
             if sys.version_info > (3,):
                 if self.logging_file is not None:
                     self.log_py3(message)
+                    if self.eventlogger != None:
+                        self.eventlogger.trace_message(level, message)
                 else:
                     pass
             else:
                 self._log(self._get_log_prefix() + message)
+                if self.eventlogger != None:
+                    self.eventlogger.trace_message(level, message)
```

Comment:
better to put a try, except here to be safe (like we did in compositeLogger for windows) so that should any unexpected error occurs in trace_message it doesn't block the main run. in except we can just pass.